spec/isolator/orm_adapters/active_record_spec.rb

```diff
@@ -87,6 +87,40 @@
         end
       end
     end
+
+    context "Email sending" do
+      context "ActionMailer" do
```

Comment:
Let's add the following case:
-`:active_job` adapter is disabled
- `deliver_later` is called
- **no error** is raised with a transaction.

I want to make sure, that `action_mailer` adapter detects only delivery itself and not background job enqueueing.

Another case – email rendering without sending, i.e. `SampleEmail.hello.body` should not raise errors.

Reply:
@palkan done